test/spec/index.spec.js

```diff
@@ -14,33 +16,147 @@ describe('index.js', function() {
       var parsedOptions = krabby._getOptions(options);
       assert(_.isEqual(options, parsedOptions));
     });
+    it('finds a krabby config when found inside of another object', function() {
```

Comment:
Humm, I still don't think I understand. Is there another way to word the test that would help?